src/app/components/client/toolbar/AppPicker.module.scss

```diff
@@ -0,0 +1,132 @@
+@import "../../../tokens";
+
+.wrapper {
+  align-items: center;
+  position: relative;
+}
+
+.trigger {
+  display: flex;
+  cursor: pointer;
+  background-color: transparent;
+  border-style: none;
+  // Diameter of the Bento image:
+  width: 24px;
+  height: 24px;
+  padding: 0;
+  border-radius: $border-radius-sm;
+
+  svg {
+    color: $color-violet-70;
+
+    &:hover {
+      color: $color-violet-90;
+    }
+  }
+}
+
+.popover {
+  position: relative;
+
+  .triggerAnchor {
+    position: absolute;
+    width: 12px;
+    height: 12px;
+    background-color: $color-white;
+    transform: rotate(45deg) translateX(-50%)
+      translateY(calc(50% + $spacing-sm / 2));
+  }
+
+  .popup {
+    margin-top: $spacing-sm;
+    padding: 0;
+    padding-bottom: $spacing-lg;
+    width: $content-xs;
+    list-style-type: none;
+    box-shadow: $box-shadow-sm;
+    border-radius: $border-radius-sm;
+    background-color: $color-white;
+
+    .appPickerHeading {
+      color: $color-black;
+      padding: $spacing-lg;
+      text-align: center;
+      display: flex;
+      flex-direction: row;
+      align-items: center;
+      gap: $spacing-sm;
+      background-image: linear-gradient(
+          to bottom,
+          $color-white calc(100% - 3px),
+          rgba(0, 0, 0, 0)
```

Comment:
```suggestion
          rgba($color-black, 0)
```

Reply:
I don't really want to imply that the colour here matters, since it's fully transparent anyway. But it could use a comment to clarify what I'm actually doing here, since it's somewhat esoteric: 82ac529b3